lnwallet/chanfunding/coin_select_test.go

```diff
@@ -165,18 +168,18 @@ func TestCoinSelect(t *testing.T) {
 				{
 					TxOut: wire.TxOut{
 						PkScript: p2wkhScript,
-						Value:    1 * btcutil.SatoshiPerBitcoin,
```

Comment:
question: Should Code Formatting normally be done in a separate commit, I remember reading it in the guidline but cannot find it anymore ?

Reply:
Yes, will revert the format changes.